src/utils.ts

```diff
@@ -52,6 +66,23 @@ export namespace Utils {
         return ingressHost
     }
 
+    export async function getRuntimeVersion(headers: Record<string, string>, runtimeName?: string , platformHost = 'https://g.codefresh.io'): Promise<string> {
```

Comment:
it's a duplication of existing method `getRuntimeIngressHost`, let's merge them.